.github/workflows/run-in-renode.yml

```diff
@@ -1,28 +1,52 @@
+
+# !! WARNING !!
+# Do not modify this file directly!
+# !! WARNING !!
+#
+# It is generated from: ../workflows-src/run-in-renode.yml
+# using the script from https://github.com/mithro/actions-includes@main
+
 name: Run in Renode
 on: [push, pull_request, workflow_dispatch]
 jobs:
-  run-in-renode:
+  proj_template:
     runs-on: ubuntu-20.04
     steps:
-      - run: wget -O- https://static.dev.sifive.com/dev-tools/riscv64-unknown-elf-gcc-8.3.0-2020.04.1-x86_64-linux-ubuntu14.tar.gz | tar -xzC /opt
-      - run: echo "/opt/riscv64-unknown-elf-gcc-8.3.0-2020.04.1-x86_64-linux-ubuntu14/bin" >> $GITHUB_PATH
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
-        with:
-          python-version: '3.7'
-      - run: bash scripts/setup -ci
-      - run: pwd && source environment && cd proj/proj_template && make -j8 software && cp build/software.elf renode/
-      - name: Run tests
-        uses: antmicro/renode-actions/test-in-renode@main
-        with:
-          renode-version: '1.12.0+20210403git44d6786'
-          tests-to-run: 'proj/proj_template/renode/litex-vexriscv-tflite.robot'
-      - name: Archive results
-        uses: actions/upload-artifact@v2
-        with:
-          name: test-results
-          path: |
-            report.html
-            log.html
-            robot_output.xml
+    - name: ⏰ 🛂 📕 - Checking workflow expansion is up to date
```

Comment:
@PiotrZierhoffer does this check that I didn't forget to do the `make update` in `.github/workflows-src/` ?

Reply:
That's correct. Actually, this step is added automatically by actions-include

Reply:
> That's correct. Actually, this step is added automatically by actions-include

That's nifty!